lib/yarn.js

```diff
@@ -32,6 +34,29 @@ const yarn = {
         passages[pid].rawIntents.push(ix)
       })
 
+      const actions = text.match(/{{.+}}/g)
+      if (actions) {
+        actions.forEach((actionText) => {
+          actionText = actionText.replace(/^{{/, '').replace(/}}$/, '')
+
+          const action = {}
+          const elements = actionText.split(':')
+          const values = elements[2].split('|')
```

Comment:
You could extract all these values with a single RegExp and deconstruct them straight into a Object.